runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -335,4 +347,39 @@ public void onVarbitChanged(VarbitChanged event)
 			trawlerNotificationSent = false;
 		}
 	}
+
+	@Subscribe
+	public void onWidgertLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.FISHING_TRAWLER_GROUP_ID)
+		{
+			endTime = Instant.now().plusSeconds(fishingTrawlerTimeLimit);
+		}
+	}
+
+	/**
+     * Changes the Fishing Trawler timer widget from minutes to minutes and seconds
+     *
+     */
+	private void setTrawlerTimer()
+	{
+		int regionID = client.getLocalPlayer().getWorldLocation().getRegionID();
+
+		if (regionID == TRAWLER_SHIP_REGION_NORMAL || regionID == TRAWLER_SHIP_REGION_SINKING)
+		{
+			Widget trawlerTimerWidget = client.getWidget(WidgetID.FISHING_TRAWLER_GROUP_ID, 37);
+
+			Instant now = Instant.now();
+			long timerInSeconds = Duration.between(now, endTime).getSeconds();
+			int minutes = (int) (timerInSeconds % 3600) / 60;
+			int seconds = (int) timerInSeconds % 60 > 0 ? (int) timerInSeconds % 60 : 0;
+
+			String trawlerText = minutes > 0 ?
+					String.format("Time Left: " + minutes + " Mins " + seconds + " Secs") :
```

Comment:
`String.format("Time Left: %02d Mins %02d Secs", minutes, seconds);`

Reply:
I'd honestly recommend just concatenating strings or using a string builder for this instead since the "%02d Mins " portion shouldn't be added when `minutes` is 0.

Reply:
I think it should be displayed like a timer, the %02d can be dropped. I'm not sure how it was shown by default. 

They are also dropping minutes when it's done.

Reply:
The default is "Time Left: 12 Mins". I tried to keep it consistent with the original.